x-pack/metricbeat/module/panw/routing/_meta/fields.yml

```
@@ -0,0 +1,158 @@
+- name: routing
+  type: group
+  release: beta
+  description: >
+    routing
+  fields:
+    - name: bgp.peer_name
+      type: keyword
+      description: >
+        The name of the current peer in the BGP peer group
+    - name: bgp.virtual_router
+      type: keyword
+      description: >
+        The virtual router with which the BGP peer is associated
+    - name: bgp.peer_group
+      type: keyword
+      description: >
+        The name of the BGP peer group this peer belongs to
+    - name: bgp.peer_router_id
+      type: ip
+      description: >
+        BGP peer router ID
+    - name: bgp.remote_as_asn
+      type: long
+      description: >
+        The remote Autonomous System (AS) number of the peer
+    - name: bgp.status
+      type: keyword
+      description: >
+        The BGP session status, e.g., "Established" means the session is up and running
+    - name: bgp.status_duration.sec
+      type: long
+      description: >
+        Time in seconds since the current status was set
+    - name: bgp.password_set
+      type: boolean
+      description: >
+        Indicates whether a password is set for the BGP peer
+    - name: bgp.passive
+      type: boolean
+      description: >
+        Indicates if the BGP peer is in passive mode: if yes then router will not initiate a connection to the peer
+    - name: bgp.multi_hop_ttl
+      type: long
+      description: >
+        Time to Live (TTL) value for multi-hop BGP sessions. Units are the number of hops.
+    - name: bgp.peer_ip
+      type: ip
+      description: >
+        IP address of the peer
+    - name: bgp.peer_port
+      type: long
+      description: >
+        Port number of the peer
+    - name: bgp.local_ip
+      type: ip
+      description: >
+        Local ip address used for BGP connection
+    - name: bgp.local_port
+      type: long
+      description: >
+        Local port number used for BGP connection 
+    - name: bgp.reflector_client
+      type: keyword
```

```suggestion
      type: bool
```

Based on description, looks like a bool?

---

` <reflector-client>not-client</reflector-client>`

---

Got it. Thanks!